### DISCUSSION TITLE
Linear Static Implicit Computation in OpenRadioss

### DISCUSSION THREAD

**Discussion by JAWilliams123** [OpenRadioss/OpenRadioss#3149] [Q&A] [answered]
Dear All,
I have been able to compile the implicit solver for OpenRadioss on windows and can run non-linear simulations using /IMPL/NONLIN. My issue is that when running /IMPL/LINEAR as part of a /IMPL/BUCKL simulation, I receive error 8683 Fatal error: MUMPS required. The question here is whether or not /IMPL/LINEAR and/or /IMPL/BUCKL have been disabled in OpenRadioss? If it has been enabled, what flags need to be enabled to allow for compilation of the required bits to move things forward?

Many thanks,
JAWilliams

**Comment by laurent-altr** [comment, marked as answer]
Hi @JAWilliams123 , 

Thank you for your inquiry. Unfortunately, /IMP/BUCKL is not currently available in OpenRadioss. It is one of the few options we do not support, alongside /EIG, /FVMBAG, /INIMAP2D, and others.

The reasons for this may include:
-  Dependence on third-party libraries that are incompatible with our open-source license.
-  Incompatibility with free compilers, such as gfortran.
-  Deprecation of features or functionality that no longer work as intended.
-  ...

I do not know if there are plans to release this option in OpenRadioss. My understanding is that doing so would require non-trivial changes to the codebase. Perhaps others in the development team may be able to provide additional insights or comments on this matter.

Thank you, 
Laurent

**Reply by JAWilliams123** [reply]
Hello @laurent-altr,

Many thanks for the follow-up. 

Best wishes,
JAWilliams
